### DISCUSSION TITLE
When did the WSLInterop binfmt flags change from F to PF?

### DISCUSSION THREAD

**Discussion by cerebrate** [microsoft/WSL#8566] [Q&A]
This came up as a `genie` bug for me (https://github.com/arkane-systems/genie/issues/267, and https://github.com/arkane-systems/genie/issues/264) but, of course, fixing it for later versions breaks it for earlier versions.

Is information available on exactly when (by, say, build number so I can check it programmatically) this change was made so I can keep supporting all Windows/WSL versions?

Thanks in advance.

**Comment by Biswa96** [comment]
Can't you check the actual binfmt file `/proc/sys/fs/binfmt_misc/WSLInterop` instead of checking Windows version? BTW, Windows 10 Build 19044 does **not** have that P flag. Maybe, it's from Win11.

**Reply by cerebrate** [reply]
That's my fallback option (which I'm implementing _right now_ :grin:), but I'd prefer to go off what it "ought" to be rather than rely on a version that might already have been changed.
